### DISCUSSION TITLE
Plex user vs plex friends issue

### DISCUSSION THREAD

**Discussion by wasifab** [sct/overseerr#3557] [Help]
I have overseerr installed on my windows machine via Docker Desktop. I followed the documentation install instructions and it seems like i'm getting a common issue from previous versions where Overseerr can only see Plex "Friends" not all plex "users" with library access. 

I'm aware that this issue #3550 was solved in the last week but i can't seem to fix this on my install which was pulled a few days after this fix was committed. 

Any help is appreciated.

**Comment by AbaWasif** [comment]
I've managed to find the plextv.ts file in my docker image where this issue should have been fixed and it looks like it hasn't been updated to reflect the latest commit. I guess i'll just wait till the docker image has also been updated.
